lib/shared/config-manager/src/index.ts

```diff
@@ -113,6 +128,19 @@ export class EnvironmentConfigManager {
             }
         }
 
+        const trackEvent = (err?: ResponseError) => {
+            if ((res && res?.status !== 304) || err) {
+                this.trackSDKConfigEvent(
+                    url,
+                    responseTimeMS,
+                    res || undefined,
+                    err,
+                    reqEtag,
```

Comment:
shouldn't we be tracking the received etag and last modified in this event?

Reply:
yea that's taken from the `res` object.